enterprise/server/cmd/ci_runner/main.go

```diff
@@ -762,6 +762,8 @@ func run() error {
 // to be called after the current invocation has completed, to avoid blocking
 // the invocation status from being reported.
 func (ws *workspace) prepareRunnerForNextInvocation(ctx context.Context, taskWorkspaceDir string) {
+	log := ws.log
+
```

Comment:
nit: I'd prefer if you just explicitly use ws.log below to stay consistent with the rest of the file